IQKeyboardManager/IQKeyboardManager.m

```diff
@@ -420,11 +420,6 @@ -(void)setEnableAutoToolbar:(BOOL)enableAutoToolbar
     {
         [self addToolbarIfRequired];
     }
-    //Else removing toolbar.
-    else
-    {
-        [self removeToolbarIfRequired];
```

Comment:
Suppose if user changes enableAutoToolbar property in the middle, then the library should also remove previously added toolbar. That's why we call [self removeToolbarIfRequired];